learning/func.py

```diff
@@ -0,0 +1,179 @@
+import io
+import os
+import json
+import logging
+import aiohttp
+import asyncio
+import feedparser
+import oci
+import datetime
+import uuid
+import fdk
+
+from fdk import response
```

Comment:
Do we have a style guide? Eg, what level to specify import (module or class), etc.

Reply:
Not currently. Anyone you like in particular?